rdm/cli.py

```diff
@@ -27,14 +30,22 @@ def init(output_directory):
 def parse_arguments(arguments):
     parser = argparse.ArgumentParser(prog='rdm')
     subparsers = parser.add_subparsers(dest='command', metavar='<command>')
+
     init_help = 'initialize a set of templates in the output directory'
     init_parser = subparsers.add_parser('init', help=init_help)
     init_parser.add_argument('-o', '--output', default='regulatory')
+
     render_help = 'render a template using the specified data files'
     render_parser = subparsers.add_parser('render', help=render_help)
     render_parser.add_argument('template')
     render_parser.add_argument('data_files', nargs='*')
+
     tex_help = 'translate a yaml+gfm file into a tex file using pandoc'
     tex_parser = subparsers.add_parser('tex', help=tex_help)
     tex_parser.add_argument('input')
+
+    requirements_help = 'pull requirements and problem reports from project management tools'
+    requirements_parser = subparsers.add_parser('pull', help=requirements_help)
```

Comment:
For consistency, I think we should name these variables `pull_help` and `pull_parser` instead of `requirements_help` and `requirements_parser`.  Alternatively, we could change the subcommand from `pull` to `requirements`.

Reply:
oops, that was a holdover-- the subcommand was originally called `requirements`.